content/docs/user-guide/troubleshooting.md

```diff
@@ -28,3 +28,23 @@ To solve this, it's often possible to increase the open file descriptors limit,
 with `ulimit` on UNIX-like system (for example `ulimit -n 1024`), or
 [increasing Handles limit](https://blogs.technet.microsoft.com/markrussinovich/2009/09/29/pushing-the-limits-of-windows-handles/)
 on Windows. Otherwise, please try using a lower `JOBS` value.
+
+## Unable to find credentials {#no-credentials}
```

Comment:
```suggestion
## Unable to find credentials {#no-aws-credentials}
```

Reply:
Never mind about this one (per https://github.com/iterative/dvc/pull/4480#pullrequestreview-476372465).